src/events/util.js

```diff
@@ -12,4 +12,15 @@ export default {
     const recurrence = recurrences[event.recurringType];
     return `${i18n.t(recurrence)} ${i18n.t('on')} ${dayName}`;
   },
+  getNextStartTime(event) {
+    const now = moment();
+    const nextDateInfo = event.dates.find((dateInfo) => {
+      const dateMoment = moment(dateInfo.startTime);
+      return dateMoment.isAfter(now);
```

Comment:
I know you just moved this code from one place to another, but it's going to have the same issue as outlined in https://github.com/CoderDojo/cp-zen-frontend/pull/13#discussion_r129610388